backend/src/main/modules/helper.py

```diff
@@ -0,0 +1,1010 @@
+import os
+import time
+from configparser import ConfigParser, ExtendedInterpolation
+import xlrd
+import uuid
+import csv
+from bson.objectid import ObjectId
+import json
+from datetime import datetime
+import requests
+from difflib import get_close_matches
+from requests import post, get, delete
+import sys
+import time
+import shutil
+from xlutils.copy import copy
+import shutil
+import re
+from xlrd import open_workbook
+from xlutils.copy import copy as xl_copy
+import logging.handlers
+import time
+from logging.handlers import TimedRotatingFileHandler
+import xlsxwriter
+import argparse
+import sys
+from os import path
+import openpyxl
+from openpyxl import Workbook, load_workbook
+from openpyxl.styles import Color, PatternFill, Font, Border
+from openpyxl.styles import colors
+from openpyxl.cell import Cell
+from common_config import *
+import threading
+
+millisecond = None
+childsolutionid = ""
+regex = "\"?([-a-zA-Z0-9.`?{}]+@\w+\.\w+)\"?"
+class Helpers:
+    def __init__(self):
+        self.millisecond = None
+
+    def programCreation(accessToken, parentFolder, externalId, pName, orgIds,creatorKeyCloakId, creatorName):
+        global programIdForSuccess, programExternalId
+       
+        programCreationurl = internal_kong_ip_core +  programcreationurl
+        
+        programExternalId = externalId
+        payload = json.dumps({
+            "externalId": externalId,
+            "name": pName,
+            "description": "Program",
+            "resourceType": [
+                "program"
+            ],
+            "language": [
+                "English"
+            ],
+            "keywords": [],
+            "concepts": [],
+            "createdFor": orgIds,
+            "rootOrganisations": orgIds,
+            "startDate": startDateOfProgram,
+            "endDate": endDateOfProgram,
+            "imageCompression": {
+                "quality": 10
+            },
+            "creator": creatorName,
+            "owner": creatorKeyCloakId,
+            "author": creatorKeyCloakId,
+            # "metaInformation": {
+            #     # "state":entitiesPGM.split(","),
+            #     # "role": mainRole.split(",")
+            #     },
+                "requestForPIIConsent":False
+            })
+
+        
+        headers = {'X-authenticated-user-token': accessToken,
+                   'internal-access-token': internal_access_token,
+                   'Content-Type': 'application/json',
+                   'Authorization':authorization}
+
+        # program creation 
+        responsePgmCreate = requests.request("POST", programCreationurl, headers=headers, data=(payload))
+        print(responsePgmCreate.text)
```

Comment:
id this required here? If needed create a logger file and add ur logs there. it doesnt make any sense in showing the logs in terminal. it won't be preserved. 
While creating log, make sure its a rotational log